File: HELPS/ClientApp/src/types/model/Advisor.ts
export interface AdvisorVariable {
    firstName: string,
    lastName: string,
    staffNumber: number,
    email: string,
    status: string
}

export interface Advisor {
    id: number,
    email: string,
    firstName: string,
    lastName: string,
    isActive: boolean,
    variables: AdvisorVariable[]
}

File: HELPS/Models/Advisor.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace HELPS.Models
{
    public class Advisor
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public bool IsActive { get; set; }
    }
}